docs/source/containerization/docker.mdx

```diff
@@ -58,18 +58,32 @@ docker run -p 4000:4000 \
   --rm \
   --interactive \
   --tty \
-  --entrypoint=sh
-  ghcr.io/apollographql/router:<image version>-debug
-/dist # pwd
+  --entrypoint=bash \
+  ghcr.io/apollographql/router:<image version>
+root@0c6ae96a2bbd:/dist# pwd
 /dist
-/dist # ls
-LICENSE        config         router
-README.md      licenses.html  schema
-/dist # exit
+root@0c6ae96a2bbd:/dist# ls
+config	router	schema
+root@0c6ae96a2bbd:/dist# exit
```

Comment:
same

Reply:
done in: https://github.com/apollographql/router/pull/2142/commits/2bd3a4e0e4758a470e6351749f2fb3d2598bf376